sdk/communication/Azure.Communication.CallAutomation/CHANGELOG.md

```diff
@@ -10,6 +10,54 @@
 
 ### Other Changes
 
+## 1.4.0-beta.1 (2024-11-22)
```

Comment:
(2024-11-22) is the same date for 1.3.0 line 24?

Reply:
Also do we have InterruptAudioAndAnnounce api here? and all new events for hold and play?

Reply:
> (2024-11-22) is the same date for 1.3.0 line 24?

Yes GA4 and Beta5 are released on the same date.

Reply:
> Also do we have InterruptAudioAndAnnounce api here? and all new events for hold and play?

we have the generated model changes, but needs to be updated in the SDK but will do it another PR.